diff --git a/esm/graphql-config.js b/esm/graphql-config.js
index 31e151c219846fde627b908b89106ad00a22601a..ce642710a4a82bac3336c806aa14745ef92acd7c 100644
--- a/esm/graphql-config.js
+++ b/esm/graphql-config.js
@@ -1,7 +1,6 @@
 import fs from "node:fs";
 import path from "node:path";
 import debugFactory from "debug";
-import { GraphQLConfig, loadConfigSync } from "graphql-config";
 import { CodeFileLoader } from "@graphql-tools/code-file-loader";
 const debug = debugFactory("graphql-eslint:graphql-config");
 let graphQLConfig;
