diff --git a/rules/no-unnecessary-polyfills.js b/rules/no-unnecessary-polyfills.js
index b58183018e4d07143c032e0f1686835437c25391..26f5687e0c9eb3bb4d0b74f9a5e9736990c7d7ab 100644
--- a/rules/no-unnecessary-polyfills.js
+++ b/rules/no-unnecessary-polyfills.js
@@ -1,10 +1,9 @@
 import path from 'node:path';
-import coreJsCompat from 'core-js-compat';
 import {camelCase} from 'change-case';
 import isStaticRequire from './ast/is-static-require.js';
 import {readPackageJson} from './shared/package-json.js';
 
-const {data: compatData, entries: coreJsEntries} = coreJsCompat;
+const {data: compatData, entries: coreJsEntries} = { data: {}, entries: {} };
 
 const MESSAGE_ID_POLYFILL = 'unnecessaryPolyfill';
 const MESSAGE_ID_CORE_JS = 'unnecessaryCoreJsModule';
