diff --git a/lib/node-resolver.js b/lib/node-resolver.js
index d441e68a3fe8a0cacb5f5aa997ed57732f62bc0d..b3cbf13b75eda9253865f41ac575e2f5cf299e0e 100644
--- a/lib/node-resolver.js
+++ b/lib/node-resolver.js
@@ -1,6 +1,5 @@
 import { isBuiltin } from 'node:module';
 import path from 'node:path';
-import { ResolverFactory } from 'unrs-resolver';
 export function createNodeResolver({ extensions = ['.mjs', '.cjs', '.js', '.json', '.node'], conditionNames = ['import', 'require', 'default'], mainFields = ['module', 'main'], ...restOptions } = {}) {
     const resolver = new ResolverFactory({
         extensions,
