diff --git a/lib/node-resolver.js b/lib/node-resolver.js
index 7e1c80c9652d98e10846c5289a1c484711684ccc..8c634dad8342a8eeae562cc0d78414d8f657e435 100644
--- a/lib/node-resolver.js
+++ b/lib/node-resolver.js
@@ -4,8 +4,8 @@ exports.createNodeResolver = createNodeResolver;
 const tslib_1 = require("tslib");
 const node_module_1 = tslib_1.__importDefault(require("node:module"));
 const node_path_1 = tslib_1.__importDefault(require("node:path"));
-const rspack_resolver_1 = require("rspack-resolver");
 function createNodeResolver({ extensions = ['.mjs', '.cjs', '.js', '.json', '.node'], conditionNames = ['import', 'require', 'default'], mainFields = ['module', 'main'], ...restOptions } = {}) {
+    const rspack_resolver_1 = require("rspack-resolver");
     const resolver = new rspack_resolver_1.ResolverFactory({
         extensions,
         conditionNames,
