diff --git a/src/rules/eslint.d.ts b/src/rules/eslint.d.ts
index d0d80e1..45b0542 100644
--- a/src/rules/eslint.d.ts
+++ b/src/rules/eslint.d.ts
@@ -898,10 +898,6 @@ export interface NoConsoleOption {
   allow?: [string, ...string[]];
 }
 
-export interface NoConstructorReturnOption {
-  [k: string]: any;
-}
-
 export interface NoEmptyFunctionOption {
   allow?: (
     | 'functions'
@@ -2403,7 +2399,7 @@ export interface EslintRules {
    * Disallow returning value from constructor.
    * @see [no-constructor-return](https://eslint.org/docs/latest/rules/no-constructor-return)
    */
-  'no-constructor-return': NoConstructorReturnOption;
+  'no-constructor-return': null;
 
   /**
    * Disallow `continue` statements.
