diff --git a/src/rules/graphql-eslint.d.ts b/src/rules/graphql-eslint.d.ts
index 5f890f8..7f51103 100644
--- a/src/rules/graphql-eslint.d.ts
+++ b/src/rules/graphql-eslint.d.ts
@@ -290,8 +290,8 @@ export namespace NamingConvention {
           VariableDefinition?: AsString | AsObject;
           allowLeadingUnderscore?: boolean;
           allowTrailingUnderscore?: boolean;
-
-          [k: string]: AsString | AsObject;
+        } & {
+          [k: string]: AsString | AsObject | boolean | undefined;
         },
       ];
   /**
