diff --git a/src/rules/graphql-eslint.d.ts b/src/rules/graphql-eslint.d.ts
index d6a6562..b1c44b6 100644
--- a/src/rules/graphql-eslint.d.ts
+++ b/src/rules/graphql-eslint.d.ts
@@ -259,8 +259,7 @@ export namespace NamingConvention {
           VariableDefinition?: AsString | AsObject;
           allowLeadingUnderscore?: boolean;
           allowTrailingUnderscore?: boolean;
-          /**
-           */
+        } & {
           [k: string]: AsString | AsObject;
         },
       ];
