diff --git a/src/rules/playwright.d.ts b/src/rules/playwright.d.ts
index b2b88f6..90ea40b 100644
--- a/src/rules/playwright.d.ts
+++ b/src/rules/playwright.d.ts
@@ -29,13 +29,13 @@ export interface ValidTestTagsOption {
   )[];
 }
 
-export interface ValidTitleOption {
+export type ValidTitleOption = {
   disallowedWords?: string[];
   ignoreSpaces?: boolean;
   ignoreTypeOfDescribeName?: boolean;
   ignoreTypeOfStepName?: boolean;
   ignoreTypeOfTestName?: boolean;
-
+} & {
   [k: string]:
     | string
     | [string]
