diff --git a/src/rules/playwright.d.ts b/src/rules/playwright.d.ts
index 9bbfb90..7a5791f 100644
--- a/src/rules/playwright.d.ts
+++ b/src/rules/playwright.d.ts
@@ -14,7 +14,7 @@ export interface PreferLowercaseTitleOption {
   ignoreTopLevelDescribe?: boolean;
 }
 
-export interface ValidTitleOption {
+export type ValidTitleOption = {
   disallowedWords?: string[];
   ignoreSpaces?: boolean;
   ignoreTypeOfDescribeName?: boolean;
@@ -22,6 +22,7 @@ export interface ValidTitleOption {
   ignoreTypeOfTestName?: boolean;
   /**
    */
+} & {
   [k: string]:
     | string
     | [string]
@@ -29,7 +30,7 @@ export interface ValidTitleOption {
     | {
         [k: string]: string | [string] | [string, string];
       };
-}
+};
 
 /**
  * All Playwright rules.
