diff --git a/src/rules/playwright.d.ts b/src/rules/playwright.d.ts
index 2386749..05b23b7 100644
--- a/src/rules/playwright.d.ts
+++ b/src/rules/playwright.d.ts
@@ -35,15 +35,8 @@ export interface ValidTitleOption {
   ignoreTypeOfDescribeName?: boolean;
   ignoreTypeOfStepName?: boolean;
   ignoreTypeOfTestName?: boolean;
-  /**
-   */
-  [k: string]:
-    | string
-    | [string]
-    | [string, string]
-    | {
-        [k: string]: string | [string] | [string, string];
-      };
+  mustNotMatch?: Partial<Record<'describe' | 'test', string>> | string;
+  mustMatch?: Partial<Record<'describe' | 'test', string>> | string;
 }
 
 /**
