diff --git a/src/rules/vitest.d.ts b/src/rules/vitest.d.ts
index 09b9fa7..85f2dec 100644
--- a/src/rules/vitest.d.ts
+++ b/src/rules/vitest.d.ts
@@ -28,10 +28,11 @@ export interface NoRestrictedMatchersOption {
   [k: string]: string | null;
 }
 
-export interface ValidTitleOption {
+export type ValidTitleOption = {
   ignoreTypeOfDescribeName?: boolean;
   allowArguments?: boolean;
   disallowedWords?: string[];
+} & {
   /**
    */
   [k: string]:
@@ -41,7 +42,7 @@ export interface ValidTitleOption {
     | {
         [k: string]: string | [string] | [string, string];
       };
-}
+};
 
 /**
  * All Vitest rules.
