diff --git a/src/rules/vitest.d.ts b/src/rules/vitest.d.ts
index 9c5015f..3864fd9 100644
--- a/src/rules/vitest.d.ts
+++ b/src/rules/vitest.d.ts
@@ -38,12 +38,11 @@ export interface NoStandaloneExpectOption {
   [k: string]: any;
 }
 
-export interface ValidTitleOption {
+export type ValidTitleOption = {
   ignoreTypeOfDescribeName?: boolean;
   allowArguments?: boolean;
   disallowedWords?: string[];
-  /**
-   */
+} & {
   [k: string]:
     | string
     | [string]
@@ -51,7 +50,7 @@ export interface ValidTitleOption {
     | {
         [k: string]: string | [string] | [string, string];
       };
-}
+};
 
 /**
  * All Vitest rules.
