diff --git a/src/rules/vitest.d.ts b/src/rules/vitest.d.ts
index 1082ad2..0544191 100644
--- a/src/rules/vitest.d.ts
+++ b/src/rules/vitest.d.ts
@@ -33,12 +33,11 @@ export interface NoRestrictedMatchersOption {
   [k: string]: string | null;
 }
 
-export interface ValidTitleOption {
+export type ValidTitleOption = {
   ignoreTypeOfDescribeName?: boolean;
   allowArguments?: boolean;
   disallowedWords?: string[];
-  /**
-   */
+} & {
   [k: string]:
     | string
     | [string]
@@ -46,7 +45,7 @@ export interface ValidTitleOption {
     | {
         [k: string]: string | [string] | [string, string];
       };
-}
+};
 
 /**
  * All Vitest rules.
