diff --git a/src/rules/vitest.d.ts b/src/rules/vitest.d.ts
index 28caf63..3a72259 100644
--- a/src/rules/vitest.d.ts
+++ b/src/rules/vitest.d.ts
@@ -31,20 +31,13 @@ export interface NoRestrictedMatchersOption {
   [k: string]: string | null;
 }
 
-export interface ValidTitleOption {
+export type ValidTitleOption = {
   ignoreTypeOfDescribeName?: boolean;
   allowArguments?: boolean;
   disallowedWords?: string[];
-  /**
-   */
-  [k: string]:
-    | string
-    | [string]
-    | [string, string]
-    | {
-        [k: string]: string | [string] | [string, string];
-      };
-}
+} & {
+  [k: string]: any;
+};
 
 /**
  * All Vitest rules.
