diff --git a/src/rules/vitest.d.ts b/src/rules/vitest.d.ts
index 4ae61d7..038bb6f 100644
--- a/src/rules/vitest.d.ts
+++ b/src/rules/vitest.d.ts
@@ -36,14 +36,8 @@ export interface ValidTitleOption {
   ignoreTypeOfDescribeName?: boolean;
   allowArguments?: boolean;
   disallowedWords?: string[];
-
-  [k: string]:
-    | string
-    | [string]
-    | [string, string]
-    | {
-        [k: string]: string | [string] | [string, string];
-      };
+  mustNotMatch?: string[];
+  mustMatch?: string[];
 }
 
 /**
